bin/test.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+let Promise = require('bluebird');
+let spawn = require('child_process').spawn;
+let fs = Promise.promisifyAll(require('fs'));
+let path = require('path');
+
+let cmd = path.resolve(__dirname, '..', 'node_modules', '.bin', '_mocha');
+let args = ['--compilers', 'ts:ts-node/register,tsx:ts-node/register'];
+
+let services = path.resolve('services');
+return fs.readdirAsync(services).then(folders => {
+  return Promise.each(folders, f => {
+    let dir = path.resolve(services, f);
+    return fs
+      .readdirAsync(dir)
+      .then(subfolders => {
+        if (subfolders.indexOf('test') > 0) {
+          return new Promise((resolve, reject) => {
+            console.log('RUN TESTS - %s:', f);
+            let testArgs = args.push(path.resolve(dir, 'test/**/*test.ts'));
+            let test = spawn(cmd, args);
+            test.stdout.on('data', out => {
+              console.log(out.toString());
+            });
+
+            test.stderr.on('data', out => {
+              console.log(out.toString());
```

Comment:
The way you log [stdout](https://github.com/strongloop/loopback-next-example/pull/64/files#diff-b0c98254eb85c26670d6f414ac4758d1R29) and stderr right now, there is no way to distinguish between (unless obviously the stderr has some sort of **Error** tag on it).

You don't necessarily have to append **Error** upon logging stderr, it could be something else (i.e `stderr`) as long as we can differentiate the stdout and stderr.